angularjs-portal-frame/src/main/webapp/frame.html

```diff
@@ -53,26 +53,7 @@
 
   <!-- FOOTER  -->
   <site-footer></site-footer>
-<script type="text/javascript" src="js/config.js"></script>
-<script type="text/javascript" src="bower_components/jquery/dist/jquery.min.js"></script>
-<script type="text/javascript" src="bower_components/jquery-ui/jquery-ui.min.js"></script>
-<script type="text/javascript" src="bower_components/angular/angular.min.js"></script>
-<script type="text/javascript" src="bower_components/angular-route/angular-route.min.js"></script>
-<script type="text/javascript" src="bower_components/angular-sanitize/angular-sanitize.min.js"></script>
-<script type="text/javascript" src="bower_components/ngstorage/ngStorage.min.js"></script>
-<script type="text/javascript" src="js/sortable.js"></script>
-<script type="text/javascript" src="js/angular-page.js"></script>
-<script type="text/javascript" src="js/main/main-controllers.js"></script>
-<script type="text/javascript" src="js/main/search-controller.js"></script>
-<script type="text/javascript" src="js/main/main-service.js"></script>
-<script type="text/javascript" src="js/main/main-directives.js"></script>
-<script type="text/javascript" src="js/app-config.js"></script>
-<script type="text/javascript" src="js/controllers.js"></script>
-<script type="text/javascript" src="js/services.js"></script>
-<script type="text/javascript" src="js/directives.js"></script>
-<script type="text/javascript" src="js/filters.js"></script>
-<script type="text/javascript" src="js/ui-bootstrap.js"></script>
-<script type="text/javascript" src="js/ga.js"></script>
-<script type="text/javascript" src="bower_components/requirejs/require.js" data-main="main.js"></script>
+  <script type="text/javascript" src="js/ga.js"></script>
```

Comment:
The thing about ga.js is that it already does its own asynchronous loading, so putting it inside the asynchronous loader of require.js is redundant / requires initialization configuration.
Because it's mainly being called from markup rather than other scripts, and it's the only one in this situation, I figured there's nothing really to be gained by moving it to Require.